multigeiger/multigeiger.ino

```diff
@@ -754,25 +751,21 @@ void DisplayStartscreen(void) {
 
   u8x8.clear();
   #if CPU == STICK
-  // Display is only 4 lines by 8 characters; lines counting from 2 to 5
-  u8x8.setFont(u8x8_font_5x8_f);        // use really small font
-  for (int i = 2; i < 6; i++) {
-    u8x8.drawString(0, i, "        ");  // clear all 4 lines
-  }
+  // Display is only 4 lines by 8 characters; lines counting from 2 to 5.
+  snprintf(line, 9, "%s", VERSION_STR);  // 8 chars + \0 termination
+  u8x8.setFont(u8x8_font_5x8_f);
   u8x8.drawString(0, 2, "Geiger-");
   u8x8.drawString(0, 3, " Counter");
   u8x8.drawString(0, 4, "Version:");
-  sprintf(line, "%d.%d.%d", VERSION_MAJOR, VERSION_MINOR, VERSION_PATCH);
   u8x8.drawString(0, 5, line);
   #else
+  // Display is only H lines by W characters; lines counting from A to B. -- TODO
+  snprintf(line, 15, "%s", VERSION_STR);  // 14 chars + \0 termination
   u8x8.setFont(u8x8_font_7x14_1x2_f);
-  u8x8.println("Geiger-Counter");
-  u8x8.println("==============");
-  sprintf(line, "V%d.%d.%d", VERSION_MAJOR, VERSION_MINOR, VERSION_PATCH);
-  u8x8.setCursor(7 - strlen(line) / 2, 4);
-  u8x8.print(line);
-  u8x8.setCursor(1, 6);
-  u8x8.print("Info:boehri.de");
+  u8x8.drawString(0, 2, "Geiger-Counter");
+  u8x8.drawString(0, 3, "==============");
+  u8x8.drawString(0, 4, line);
+  u8x8.drawString(0, 6, "Info:boehri.de");
   #endif
```

Comment:
be consistent with the code for the small display: use drawString for the bigger display also.